embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/lifecycle/EmbraceProcessStateService.kt

```diff
@@ -125,6 +125,12 @@ internal class EmbraceProcessStateService(
         return sessionOrchestrator != null
     }
 
+    override fun sessionUpdated() {
```

Comment:
Why route this through `ProcessStateService`? `SessionOrchestrator` already has a functionally equivalent check on whether the app is in the background. IMO we can just use that (and possibly rename it from `reportBackgroundActivityStateChange` to something like `onDataCaptured`)